lib/index.js

```diff
@@ -15,8 +15,9 @@ exports.register = (server, options, next) => {
     }
 
     const rate = routeSettings.rate ? routeSettings.rate(request) : options.defaultRate(request);
+    const rateLimitKey = routeSettings.rateLimitKey ? routeSettings.rateLimitKey : options.rateLimitKey;
```

Comment:
That sends it over 120 chars, should I make it an if statement instead of ternary?